src/test/resources/RegressionTests.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="RegressionTests" parallel="methods" thread-count="4">
+    <test name="Regression" preserve-order="false">
+        <parameter name="baseURL" value="https://qa.koel.app/"/>
```

Comment:
If you want to use parameter baseURL at suite file => In the BaseTest class you need you annotations `@Parameters`
//HINT: 
![image](https://github.com/rhieya416/A49-QA-Automation-Framework/assets/144403954/4864c0f8-0ec5-4df9-af56-540b1e0299bd)